src/test/java/org/cactoos/io/TeeOutputTest.java

```diff
@@ -50,83 +50,70 @@
     @Test
     public void copiesContent() {
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        final ByteArrayOutputStream copy = new ByteArrayOutputStream();
-        MatcherAssert.assertThat(
+        new Assertion<>(
             "Can't copy Output to Output and return Input",
-            new TextOf(
-                new TeeInput(
-                    new InputOf("Hello, товарищ!"),
-                    new TeeOutput(
-                        new OutputTo(baos),
-                        new OutputTo(copy)
+            () -> new TextOf(
+                new Sticky(
```

Comment:
@Iprogrammerr Is necessary use the Sticky decorator here?

Reply:
@fabriciofx Same as above.

Reply:
@Iprogrammerr the same above.